dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1994,6 +1994,7 @@
       "properties": {
         "sentenceCount": {
           "type": "integer",
+          "format": "int32",
```

Comment:
Shouldn't this be removed as well, or was it there from the start?

Reply:
Removed.